h2o-hadoop/scripts/docker/build.py

```diff
@@ -0,0 +1,89 @@
+#! /usr/bin/python
+
+import argparse
+import subprocess
+import sys
+import os
+
+DISTRIBUTION_CDH = 'CDH'
+DISTRIBUTION_HDP = 'HDP'
+SUPPORTED_DISTRIBUTIONS = [DISTRIBUTION_CDH, DISTRIBUTION_HDP]
+
+SUPPORTED_CDH_VERSIONS = ['5.4', '5.5', '5.6', '5.7', '5.8', '5.10']
+SUPPORTED_HDP_VERSIONS = ['2.2', '2.3', '2.4', '2.5', '2.6']
+
+SUPPORTED_SPARK_VERSIONS = [
```

Comment:
I am still not fan of hardcoding version of spark here. What if i would like to test something with nightly spark?